packages/react-components/.env.example

```diff
@@ -0,0 +1,9 @@
+AWS_ACCESS_KEY_ID=xxxxxxxxxx
```

Comment:
instruction/guide to setup the env?

Reply:
+1

Reply:
yup, these are the variables you need to run that story!